index.js

```diff
@@ -28,68 +34,103 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+function drinkWater(thirstLevel) {
+  console.log("Man I sure am thirsty")
+  thirstLevel -= 1
+  console.log("Ahh that hits the spot")
+  return thirstLevel
+}
+
+
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red")
+
+function sameSameButDifferent(num1, num2){
+  if (num1 === num2) {
+    return "same same"
+  } else if (num1 == num2) {
```

Comment:
This works, but you can also convert the string to a number with `parseInt` to avoid `==`:

```js
} else if (num === parseInt(maybeNum)) {
```